shared/public/ShaderSimpleLineStyle.h

```diff
@@ -12,6 +12,9 @@ struct __attribute__((packed)) ShaderSimpleLineStyle final {
     HalfFloat widthAsPixel;
     HalfFloat opacity;
     HalfFloat lineCap;
+#ifndef __APPLE__
+    // Ensure structure size is multiple of 16 bytes due to OpenGL uniform buffer padding rules (std140)
+#endif
```

Comment:
this is probably not needed? @maurhofer-ubique

Reply:
I left it in so that the condition surely isn't missed when updating the structure. Probably the comments in OpenGL shader would be sufficient though.